wlct/cogs/clot.py

```diff
@@ -604,9 +618,11 @@ async def linkd(self, ctx, arg="invalid_cmd", arg2="invalid_id"):
             elif total_successfully_updated and arg == "-r":
                 await ctx.send("You've removed the link to this channel for {} out of {} tournaments and {} filters.".format(
                     total_successfully_updated, len(tournaments), total_filters_removed))
-        except:
-            log_exception()
-            await ctx.send("An error has occurred and was unable to process the command.")
+        except Exception as e:
+            handle_command_exception(ctx)
+
+            # Outputs error message to discord for user context
+            await ctx.send("An error has occurred:\n" + str(e) + "\nAsk -B#0292 or JustinR17#9950")
```

Comment:
All of these should be a helper function IMO. I would prefer as part of the main Bot class we make a:

def handle_command_exception_error(self, ctx):
    msg_channel = ctx.message.channel
    msg_info = "Channel/Server: " + msg_channel.name + "/" + msg_channel.guild.name
    msg_info += "\nUser: " + ctx.message.author.name + "#" + ctx.message.author.discriminator
    # Logs user and channel info to backend
    log_command_exception(msg_info)
    await ctx.send(....)

This is all over so we really want this logic in one place instead of duplicated everywhere.